src/tenants/common/Home/index.tsx

```diff
@@ -6,18 +6,22 @@ import Footer from '../../../features/common/Layout/Footer';
 import React from 'react';
 import tenantConfig from '../../../../tenant.config';
 import { useTranslation } from 'next-i18next';
+import {
+  LeaderBoardList,
+  TenantScore,
+} from '../../../features/common/types/leaderboard';
 
 const config = tenantConfig();
 
 interface Props {
-  leaderboard: any;
-  tenantScore: any;
+  leaderboard: LeaderBoardList | null;
+  tenantScore: TenantScore | null;
 }
 
 export default function About({ leaderboard, tenantScore }: Props) {
   const { t, ready } = useTranslation(['tenants']);
 
-  const descriptionRef = React.useRef(null);
+  const descriptionRef = React.useRef<HTMLElement>(null);
```

Comment:
`HTMLParagraphELement` probably fits better. There is a warning while using `descriptionRef` currently